ui/shared/TruncatedText.jsx

```diff
@@ -22,12 +20,13 @@ export default class TruncatedText extends React.Component {
           {text}
         </p>
         {text.length > maxLength && (
-          <p
+          <button
```

Comment:
Nit: Can you change this button and the button in AlertTableRow to a reactstrap Button component for consistentcy :)